sanic/server.py

```diff
@@ -189,6 +193,14 @@ def write_response(self, response):
                 response.output(
                     self.request.version, keep_alive,
                     self.request_timeout))
+            if self.has_log:
```

Comment:
I don't think this is necessary. If you don't want sanic's logging, just configuration logging and disable sanic related loggers.